components/Dialogs/src/MDCAlertController.m

```diff
@@ -563,7 +563,7 @@ - (void)viewWillLayoutSubviews {
   if (CGRectGetWidth(self.view.bounds) != _previousLayoutSize.width ||
       CGRectGetHeight(self.view.bounds) != _previousLayoutSize.height) {
     CGSize currentPreferredContentSize = self.preferredContentSize;
-    CGSize contentSize = CGRectStandardize(self.alertView.bounds).size;
+    CGSize contentSize = CGRectInfinite.size;
```

Comment:
Does the calculated preferred size eventually get constrained to the viewport's bounds? If so, then this change LGTM, otherwise I'd expect some consideration for the viewport bounds to be added as part of this PR.